src/main/java/com/squareup/javapoet/MethodSpec.java

```diff
@@ -198,12 +201,7 @@ public static Builder overriding(ExecutableElement method) {
     String methodName = method.getSimpleName().toString();
     MethodSpec.Builder methodBuilder = MethodSpec.methodBuilder(methodName);
 
-    methodBuilder.addAnnotation(OVERRIDE);
```

Comment:
He added `methodBuilder.addAnnotation(Override.class)` which has the same effect.